docs/designs/template-000.md

```diff
@@ -0,0 +1,9 @@
+# Title
+
+## Overview
```

Comment:
Would hints like "A concise, 5 sentences max, statement of the problem" be useful?

Reply:
Why do we want to limit this to only 5 sentences for a pkg manager support design doc? A given pkg manager ecosystem will typically be extremely complex, so a solid background feels very appropriate for this type of document.

Reply:
It also feels to me that "Overview" is something more akin to TL;DR. So I wouldn't want it to be a long thing, I want it to be something I can read in under 2min.

Reply:
I'd say "Overview" is for a brief introduction to a problem, "Context" is for a solid background.

Reply:
But for pkg manager design the overview in that case is we need support for one and it uses CLI commands X,Y,Z. Specifically with pkg managers which is a complex topic on its own I don't think a short overview will give you what you need, to understand it, one needs to read further. Also note that I'm not sure if there's a formal definitiion of overview, IOW overview can be arbitrarily long which the context complies with as well in that we're not covering the whole pkg manager ecosystem, we're only covering what makes sense for us which is still technically an overview == a subset of information necessary to back the implementation.

Reply:
Resolving this since we moved this discussion [here](https://github.com/containerbuildsystem/cachi2/pull/601#discussion_r1723220861).